### PR TITLE
feat: minor changes in cost analysis

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please see order by commits!!!!!!

### CODE REVIEW

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue [hunk 1]
```diff
@@ -138,29 +132,13 @@ watch(() => costAnalysisPageState.groupBy, (groupBy) => {
...
                                    :selected.sync="state.selectedTagsMenu"
-                                   :visible-menu.sync="state.tagsDropdownVisible"
+                                   :selection-label="$t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.TAGS')"
+                                   appearance-type="badge"
+                                   :show-delete-all-button="false"
                                    multi-selectable
+                                   selection-highlight
                                    show-select-marker
                                    @select="handleSelectTagsGroupBy"
                                    @clear-selection="handleClearTagsGroupBy"
```
Comment (yuda110) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisGroupByFilter.vue:144]: 스크린샷 2023-09-11 오후 2 44 53

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue [hunk 2]
```diff
@@ -95,13 +96,15 @@ const drawChart = () => {
...
-            valueYField: l.name,
+    props.legends.forEach((legend) => {
+        const seriesSettings: Partial<am5xy.IXYSeriesSettings> = {
+            name: legend.label as string,
+            valueYField: legend.name,
             stacked: true,
             stroke: undefined,
-        });
+        };
+        if (legend.color) seriesSettings.fill = color(legend.color);
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue:106]: nitpick: I added color util to chartHelper, so u can use it like `chartHelper.color()`

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue [hunk 3]
```diff
@@ -125,7 +128,7 @@ const drawChart = () => {
         const seriesColor = series.get('fill')?.toString();
         tooltip.label.adapters.add('text', (text, target) => {
             if (target?.dataItem?.dataContext) {
-                let value = target.dataItem?.dataContext?.[l.name];
+                let value = target.dataItem?.dataContext?.[legend.name];
```
Comment (WANZARGEN) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue:131]: nitpick: I want to recommend you to declare it as another constant like below:

```typescript
const data = target?.dataItem?.dataContext as Data;
if(data) ...
```

I think it's more readable!

**Review by WANZARGEN — APPROVED** [review]
LGTM~! I left very nitpick comments!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue (hunk 2 above)
Comment (yuda110) [reply apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisStackedColumnChart.vue:106]: wow thanks!!!!!!!

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**